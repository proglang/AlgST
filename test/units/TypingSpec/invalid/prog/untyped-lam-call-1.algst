-- FIXME: if we could get this to work, this would be great. However, this
-- conflicts with implicit resolution.
double : Int -> Int
double x = (\i -> i * 2) x
