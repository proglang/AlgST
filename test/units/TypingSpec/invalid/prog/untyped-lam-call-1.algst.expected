
4:14: error:
	 Cannot deduce type of parameter i in lambda abstraction at 4:13
	 Please provide a type annotation.